voice/bxml/verbs/playAudio.md

```diff
@@ -21,7 +21,7 @@ header is set on the response, media will not be cached.
 ### Text Content
 | Name     | Description                                                                                                                                  |
 |:---------|:---------------------------------------------------------------------------------------------------------------------------------------------|
-| audioUri | The URL of the audio file to play. May be a relative URL. <br> ⚠️ **ONLY** .wav files encoded as PCM or G711 are supported. |
+| audioUri | The URL of the audio file to play. May be a relative URL. <br> ⚠️ **ONLY** .wav files encoded as `G.711 a-law`, `G.711 μ-law`, `G.729`, `pcm_s16le` (signed, 16-bit, little-endian, PCM-encoded `.wav` file), or `G.722` are supported. |
```

Comment:
Maybe you could also specify the expected frequency for a-law/u-law files, as it is not obvious.

Reply:
`G711` is defined as having `8 kHz sampling frequency × 8 bits per sample` according to the spec, so I don't think this is necessary.

Reply:
Some softwares like ffmpeg implement G711 extensions and allow a-law and u-law files with frequencies other than 8kHz.
This is actually the default for ffmpeg, so it will by default generate a-law/u-law files with the sample rate of the source audio.